packages/foreman/rubygem-hammer_cli/rubygem-hammer_cli.spec

```diff
@@ -1,64 +1,60 @@
+# template: hammer_plugin
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name hammer_cli
-%global confdir hammer
 
-%global release 2
+%global release 3
 %global prereleasesource pre.develop
 %global prerelease %{?prereleasesource:.}%{?prereleasesource}
 
 %{!?_root_bindir:%global _root_bindir %{_bindir}}
 %{!?_root_mandir:%global _root_mandir %{_mandir}}
 %{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+%global hammer_confdir %{_root_sysconfdir}/hammer
 
-Summary: Universal command-line interface for Foreman
 Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 2.2.0
 Release: %{?prerelease:0.}%{release}%{?prerelease}%{?nightly}%{?dist}
+Summary: Universal command-line interface
```

Comment:
the `for Foreman` part was nice

Reply:
*technically* `hammer_cli` is not for Foreman, `hammer_cli_foreman` is :)

I don't mind changing it back, but we should also maybe update the gem descriptions then?